examples/config/ci/deployer.json

```diff
@@ -0,0 +1,153 @@
+{
+  "AWSProfile": "",
+  "AWSRegion": "us-east-1",
+  "AWSAvailabilityZone": "",
+  "AWSAMI": "ami-003d3d03cfe1b0468",
+  "ClusterName": "undefined",
+  "ClusterVpcID": "",
+  "AppInstanceCount": 2,
+  "AppInstanceType": "c7i.large",
+  "MetricsInstanceType": "t3.xlarge",
+  "AgentInstanceCount": 6,
+  "AgentInstanceType": "c7i.xlarge",
+  "ElasticSearchSettings": {
```

Comment:
Do we have plans to use ES? We have 20 million posts after all. My rationale here is the more we test, the higher the chance of spotting regressions in the PRs.

Reply:
What we don't have for testing ES is an index for the 20M posts. I agree we should add it at some point, but this first iteration was planned without it.

Reply:
Something for the future. ;)

Reply:
Could we track it in Jira at least?

Reply:
I'd like to have a second epic on automated load-tests at some point. Added it to business-as-usual for now: https://mattermost.atlassian.net/browse/MM-62062